scripts/install/opt/seagate/cortx/motr/libexec/motr_data_recovery.sh

```diff
@@ -117,20 +117,11 @@ is_user_root_user() {
     fi
 }
 
-# The function will return 0 if ioservice is started on local node,
+# The function will return 0 if ioservice is started on the specified node,
 # else this will return 1
-is_ios_running_on_local_node() {
-    if (hctl status | grep -i ioservice | grep $LOCAL_IOS_FID | grep -i started); then
-        return 0
-    else
-        return 1
-    fi
-}
-
-# The function will return 0 if ioservice is started on remote node,
-# else this will return 1
-is_ios_running_on_remote_node() {
-    if (hctl status | grep -i ioservice | grep $REMOTE_IOS_FID | grep -i started); then
+is_ios_running() {
+    local ios_fid=$1
+    if $(run_cmd "$LOCAL_NODE" "hctl status | grep -i ioservice | grep $ios_fid | grep -i started"); then
```

Comment:
Do we need to pass Node in this function? Its checking on local node only

Reply:
This is done to keep the run_cmd call uniform for executing any command on any node